src/plugins/intel_cpu/src/cpu_types.cpp

```diff
@@ -143,7 +143,7 @@ const InferenceEngine::details::caseless_unordered_map<std::string, Type> type_t
         { "GridSample", Type::GridSample},
         { "OneHot", Type::OneHot},
         { "RegionYolo", Type::RegionYolo},
-        { "Select", Type::Select},
+        { "Select", Type::Eltwise},
```

Comment:
Minor:
I'd just to suggest to move `Select` to `Eltwise` [part](https://github.com/openvinotoolkit/openvino/blob/b793e1e8ae74595584a31f1dc6ccb7e11b92e588/src/plugins/intel_cpu/src/cpu_types.cpp#L28) of list

Reply:
Done, thanks